openssl.yaml

```diff
@@ -37,6 +37,13 @@ pipeline:
       tag: openssl-${{package.version}}
       expected-commit: fb7fab9fa6f4869eaa8fbb97e0d593159f03ffe4
 
+  - name: Create dbg sourcecode
+    runs: |
+      SRCDIR=$(mktemp -d)
+      cp -r . $SRCDIR/
```

Comment:
mostly curious why copy then  move-to-dest as opposed to just copy-to-dest.

Reply:
@smoser because 

```
$ cp -r . ./subdir/
cp: cannot copy a directory, '.', into itself, './subdir/'
```

As the melange-out destination is a sub-directory of the currently checked out source code, and I chose this, than rewriting checkout/make/install to do everything in a sub-dir.

Reply:
oh. i didn't realize it was a subdir.
 thanks.